### DISCUSSION TITLE
Mill City Roasters roastpath logging not linux, will artisan work instead with MCR-1?

### DISCUSSION THREAD

**Discussion by timbo100** [artisan-roaster-scope/artisan#995] [Q&A] [answered]
Hi  we have a mill city roasters' 1kg roasting machine and they come with a "modbus" logging port.
I use linux and want to avoid needing to get something windows or what their solution (RoastPath) supports.
I also don't like cloud based solutions.
Does anyone know if the artisan software works with Artisan?    Their roasters don't appear in the supported machines.

Thanks

**Comment by fatrabbit-la** [comment, marked as answer]
I don’t know if this helps answer your question, but I also have a Mill City 1kg roaster that I’ve been using for the past 6-7 years with Artisan running on a simple Mac laptop. Rock solid.

Mill City isn’t listed in the supported machines but you can still set it up to work just fine. I use the Phidget 1048 as the interface.

Dan

Fat Rabbit Coffee
***@***.***
www.fatrabbitcoffee.com

**Reply by timbo100** [reply]
fatrabbit-la, this is helpful and encouraging.  

I'm still trying to figure out what I can assume here.  The Mill City Roaster has a USB port that is said to be connected to a PC (IOS/Windows for running roastpath) and the hardware built into the Roaster is connected to installed thermocouples.   You say you are using the Phidget 1048.  Do you connect this into the existing port or are you installing your own thermocouples? 

Thanks!!

**Reply by poundy** [reply]
Don't replace the probes or add a Phidget. Just use the USB port, however you will need to find out the modbus parameters for your specific machine. The supported machine list is all about the vendor being open with that and helping streamline the setup for owners - clearly MCR are not going to do that when they have a "competing" product. 

If I were you I'd look at the BlueKing machine setup as an example of what settings you need to locate your equivalent of...

**Comment by fatrabbit-la** [comment]
You probably have a newer machine because mine does not have a USB port built in. The existing ET and BT probes that came with the roaster go into the Phidget and a USB cable comes out of the Phidget into my computer. That’s it as far as hardware is concerned.

I can share my software settings with you but I’m not in front of the laptop I use for roasting at the moment. Is that something that would be helpful? I can send a screenshot.

Dan

Fat Rabbit Coffee
***@***.***
www.fatrabbitcoffee.com

**Comment by fatrabbit-la** [comment]
MCR has a competing product to Artisan? What product is that? I don’t see anything on their site.

I’m dedicated to Artisan but just curious.

Dan

Fat Rabbit Coffee
***@***.*** ***@***.***>
www.fatrabbitcoffee.com <http://www.fatrabbitcoffee.com/>

**Comment by MAKOMO** [comment]
The service is called [RoastPath](https://roastpath.com/) and is since early 2020 in beta. It is Windows-only, but they seem to toy recently with a macOS version.
